core/genesis.go

```diff
@@ -473,11 +472,12 @@ func (g *Genesis) toBlockWithRoot(root common.Hash) *types.Block {
 			}
 		}
 		if conf.IsPrague(num, g.Timestamp) {
-			head.RequestsHash = &types.EmptyRequestsHash
-			requests = make(types.Requests, 0)
+			emptyRequests := [][]byte{{0}}
```

Comment:
kinda weird to just have the deposit request prefix just as a bare 0 value here instead of named?